ci-operator/step-registry/idp/external-oidc/keycloak/aks/server/idp-external-oidc-keycloak-aks-server-commands.sh

```diff
@@ -0,0 +1,395 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export KUBECONFIG="${SHARED_DIR}/kubeconfig"
```

Comment:
This seems by-design automatically propagated, as https://docs.ci.openshift.org/docs/architecture/step-registry/#a-note-on-kubeconfig .